src/index.tsx

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import {App} from './App/App';
-import {BrowserRouter} from 'react-router-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter } from "react-router-dom";
+import { App } from "./App";
 
 export const Root = () => {
   ReactDOM.render(
     <>
       <BrowserRouter>
-        <App/>
+        <App />
       </BrowserRouter>
     </>,
-    document.getElementById('root')
-  )
-}
+    document.getElementById("root")
+  );
+};
 
-Root()
+Root();
```

Comment:
You can use `ReactDOM.render` without the outer function or if you want with function, then call it like `renderApp`.